include/solvers/adaptive_time_solver.h

```diff
@@ -22,7 +22,7 @@
 
 // Local includes
 #include "libmesh/system_norm.h"
-#include "libmesh/unsteady_solver.h"
```

Comment:
It might be able to, I was just being lazy.

I don't understand why `AdaptiveTimeSolver::advance_timestep()` doesn't just call `core_time_solver->advance_timestep`. This looks like the only place I would have to touch to have `AdaptiveTimeSolver` just subclass `UnsteadySolver`.